install_config/advanced_ldap_configuration/sssd_for_ldap_failover.adoc

```diff
@@ -67,6 +67,15 @@ in this topic they are kept separate.
 [[sssd-phase-1-certificate-generation]]
 == Phase 1: Certificate Generation
 
+[NOTE]
+====
+The undermentioned commands generate certificate files that will be valid for 2
+years (and 5 years for Certification authority (CA) certificate). These
+periods can be altered with `--expire-days` and `--signer-expire-days` options
+but by security reasons it is strongly recommended to not make them greater
```

Comment:
The sentence is true, because making validity period greater than default value adds more security risks than decreasing this value.